dataloader.py

```diff
@@ -58,42 +86,51 @@ def PadSequence(batch):
     It will be passed to DataLoader as the keyword argument of collate_fn.
     '''
     
+    #Get the lengths but don't do anything
+    lengths = [x["lengths"] for x in batch]
+    max_length = max(lengths)
+    lengths = torch.Tensor(lengths)
```

Comment:
Small comment: 

Looks good! just wondering though, since you implement `__len__` for all of your data loaders, wouldn't you be able to get that with `len(x)`? Could be misunderstanding what batch is.

Reply:
This length is the length of each game, i.e., how long the game lasted, while the __len__ function for the dataloader returns the number of games in the dataset. Probably I should rename the variable as "game_length" to be more clear. I need to get this length simply because I want to use it to plot the win probability graph for a single game, but since the inputs are padded in the batch, it would be a bit tedious to get the time of the game ending without knowing the length of the game.

Reply:
Yeah, I think calling this game_length would add context. No need to modify any further.